src/Plugin/pathauto/AliasType/EntityAliasTypeBase.php

```diff
@@ -179,6 +179,16 @@ protected function bulkUpdate(array $ids, array $options = array()) {
     $entities = $this->entityTypeManager->getStorage($this->getEntityTypeId())->loadMultiple($ids);
     foreach ($entities as $entity) {
       \Drupal::service('pathauto.generator')->updateEntityAlias($entity, 'bulkupdate', $options);
+
+      // Create aliases for the entity translations.
+      if ($entity->isTranslatable()) {
```

Comment:
I would expect that this can be simplifed. getTranslationLanguages() should always work, even if  it is not translatable, it will simply return only a single one. Remove that check and the FALSE, then you only need a single call to updateEntityAlias(). hasTranslation() should also not be necessay, getTranslationLanguages() will never return a translation that doesn't exist, that's exactly what it does.